apps/aeutils/src/aeu_http_client.erl

```diff
@@ -59,6 +61,12 @@ process_http_return(R) ->
 encode_get_params(#{} = Ps) ->
     encode_get_params(maps:to_list(Ps));
 encode_get_params(Params) when is_list(Params) ->
-    ["?", lists:join( "&", [ [http_uri:encode(K), "=" , http_uri:encode(V)] 
+    ["?", lists:join( "&", [ [http_uri:encode(to_string(K)), "=" , http_uri:encode(to_string(V))]
```

Comment:
good: I will later remove places where caller converts to string, such as:
https://github.com/aeternity/epoch/blob/5dc06ae79f61ca897ffc5a620a09dc7d5404f21d/apps/aeutils/src/aeu_requests.erl#L149

Reply:
👍